### PR TITLE
Adding Webpack, removing Gulp

### PR DESCRIPTION
**Pull request by simensta**
This PR address Issue #7 by adding Wepback and removing Gulp. 

I slightly modified the wepback configs from @amyrebecca's [simple-react-webpack](https://github.com/amyrebecca/simple-react-webpack) for this repo. 

@amyrebecca, @rogerhutchings: could I get your input on the work so far?

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  resolve: {
+    extensions: ['', '.js', '.jsx', '.styl'],
+    modulesDirectories: ['.', 'node_modules'],
+  },
+
+  module: {
+    preLoaders: [
+      {
+        test: /(\.js$|\.jsx$)/,
```
Comment (rogerhutchings) [inline webpack.config.js]: could be simplified to `/\.jsx?$)/` for style :P

**Comment by rogerhutchings** [comment]
Looks good to me, what do you reckon @amyrebecca

File: webpack.config.js (hunk 1 above)
Comment (simensta) [inline webpack.config.js]: done!

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        loader: 'eslint-loader',
+      },
+    ],
+    loaders: [
+      {
+        test: /\.json?$/,
+        loader: 'json',
+      },
+      {
+        test: /.jsx?$/,
```
Comment (rogerhutchings) [inline webpack.config.js]: Initial `.` needs to be escaped: `/\.jsx?$/`

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    preLoaders: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'eslint-loader',
+      },
+    ],
+    loaders: [
+      {
+        test: /\.json?$/,
```
Comment (rogerhutchings) [inline webpack.config.js]: Probably doesn't need to be a `?` here, as the `n` shouldn't be optional

**Comment by srallen** [comment]
LGTM too.

**Pull request merged**